tests/test_kinematics.py

```diff
@@ -26,7 +27,16 @@ def test_fk_mjcf():
     chain = chain.to(dtype=torch.float64)
     print(chain)
     print(chain.get_joint_parameter_names())
-    th = {'hip_1': 1.0, 'ankle_1': 1}
```

Comment:
This can get a bit annoying if people have to manually specify all joints all the time. In the test code we can demonstrate best practices with something like:
```python
th = {joint:0.0 for joint in chain.get_joint_parameter_names()}
th.update({'hip_1': 1.0, 'ankle_1': 1})
```